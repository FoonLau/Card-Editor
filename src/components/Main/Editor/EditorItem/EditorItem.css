.app-editor-item {
  margin-bottom: 10px;
  display: flex;
  align-items: center;
}

.app-editor-item label {
  width: 35%;
  text-align: right;
  margin-right: 10px;
}