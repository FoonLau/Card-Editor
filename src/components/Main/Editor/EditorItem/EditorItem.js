import React from 'react';
import './EditorItem.css'

export default ({ id, type, label, onChange, value }) => (
  <div className="app-editor-item">
    <label htmlFor={id}>{ label }</label>
    <input id={id} type={type} onChange={onChange} value={value} />
  </div>
);
