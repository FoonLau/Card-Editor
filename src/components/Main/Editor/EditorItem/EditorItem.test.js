import React from 'react';
import EditorItem from './EditorItem';
import renderer from 'react-test-renderer';

it('should render correctly', () => {
  const tree = renderer.create(<EditorItem />).toJSON();

  expect(tree).toMatchSnapshot();
});
