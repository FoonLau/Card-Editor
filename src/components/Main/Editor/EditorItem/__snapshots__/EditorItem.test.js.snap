// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="app-editor-item"
>
  <label
    htmlFor={undefined}
  />
  <input
    id={undefined}
    onChange={undefined}
    type={undefined}
    value={undefined}
  />
</div>
`;
